compose.petals.yml

```diff
@@ -3,6 +3,8 @@ services:
     container_name: cheshire_cat_core
     build:
       context: ./core/core
```

Comment:
Maybe is better to create a folder `docker` where we store all custom dockerfiles!
This way is better to debug and also know if the docker is from the core repo or local repo

Reply:
I'll refactor it into a proper folder 👍

Reply:
moved into docker folder